extensions/apps/notifications/src/components/utils/notifications-util.tsx

```diff
@@ -0,0 +1,140 @@
+    body: notification.payload.data.amsg,
+    // Title and icon for broadcast TBD decided in future iterations
+    notificationTypeIcon: <GlobeAltIcon />,
+    notificationTypeTitle: 'BROADCAST',
+    notificationAppIcon: null,
+    ctaLinkTitle: null,
+    ctaLinkUrl: null,
+    date: '',
+    isSeen: !notification.isUnread,
+    appName: '',
+  };
+
+  // Set notification type title and icon
+  switch (notification.payload.data.type) {
+    case 3:
+      returnObj.notificationTypeTitle = 'ACTIVITY';
+      returnObj.notificationTypeIcon = <BoltIcon />;
+      break;
+    case 4:
+      // Title and icon for group TBD decided in future iterations
+      returnObj.ctaLinkTitle = 'GROUP';
+      returnObj.notificationTypeIcon = <RectangleGroupIcon />;
+  }
+
+  const parsedMetaData = notification.payload.data.parsedMetaData;
+  // set CTA/button title and url to navigate
+  switch (returnObj.title) {
+    case 'New mention':
+      const beamId = (parsedMetaData.data as MentionNotificationMetaData).beamID;
+      returnObj.appName = '@akashaorg/app-antenna';
+      returnObj.ctaLinkUrl = `/beam/${beamId}`;
+      returnObj.ctaLinkTitle = 'View';
+      break;
+    case 'New reflection':
+      const reflectionId = (parsedMetaData.data as ReflectionNotificationMetaData).reflectionID;
+      returnObj.appName = '@akashaorg/app-antenna';
+      returnObj.ctaLinkUrl = `/reflection/${reflectionId}`;
+      returnObj.ctaLinkTitle = 'Go to reflection';
+      break;
+    case 'New follow':
+      const profileId = (parsedMetaData.data as FollowNotificationMetaData).follower;
+      returnObj.appName = '@akashaorg/app-profile';
+      returnObj.ctaLinkUrl = `/${profileId}`;
+      returnObj.ctaLinkTitle = 'Go to profile';
+      break;
+  }
+  const placeholderIcon =
+    // This case happens only if we sent notification from PushOrgDashboard
+    placeholderIcons[parsedMetaData?.channelIndex || ChannelOptionIndexes.ANTENNA];
+
+  returnObj.notificationAppIcon = (
+    <AppIcon
+      iconColor={{ light: 'secondaryLight', dark: 'secondaryDark' }}
+      size={{ width: 16, height: 16 }}
+      backgroundSize={32}
+      placeholderIcon={placeholderIcon}
+      background={'grey5'}
+      customStyle="min-w-[32px]"
+      solid
+    />
+  );
+
+  if (notification.timestamp) {
+    // Format notification time
+    returnObj.date = notificationFormatRelativeTime(notification.timestamp.toString());
+  }
+
+  return returnObj;
+};
+
+const notificationFormatRelativeTime = (date: string, locale?: string) => {
+  if (dayjs(date).isValid()) {
+    let time = dayjs(date);
+    if (/^[0-9]*$/.test(date)) {
+      time = date.length > 10 ? dayjs(+date) : dayjs.unix(+date);
+    }
+
+    if (locale) {
+      time = time.locale(locale);
+    }
+
+    const now = dayjs();
+    if (time.isSame(now, 'day')) {
+      // If the date is today
+      return time.fromNow();
+    } else if (time.isSame(now.subtract(1, 'day'), 'day')) {
+      // If the date is yesterday
```

Comment:
Day js also has plugins for checking today/yesterday. Might help with cleaner code. https://day.js.org/docs/en/plugin/is-yesterday